src/Components/Contributors/ContributorForm.tsx

```diff
@@ -41,12 +41,12 @@ const ContributorForm: FC<ContributorFormProps> = ({
   const [selectedUnit, setSelectedUnit] = useState<UnitSelector>(emptyUnitSelector);
   const [addDisabled, setAddDisabled] = useState(false);
 
-  async function handleSubmit(continueEditing: boolean) {
+  async function handleSubmit() {
     //TODO: finn ut om det er noen grunn til objekt-copy i det hele tatt ?
     const temp = JSON.parse(JSON.stringify(contributorData));
     const cleanedAffiliations = await handleChosenAuthorAffiliations(temp.toBeCreated.affiliations);
     temp.toBeCreated.affiliations = removeInstitutionsDuplicatesBasedOnCristinId(cleanedAffiliations);
```

Comment:
Vet du hva disse to linjene gjør? Disse burde vel blitt kalt hver gang man redigerer på affiliations i stedet for helt på slutten i forløpet.

Reply:
har ikke satt meg inn i det, nei